spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java

```diff
@@ -75,11 +76,13 @@ public void beforeTest() {
 			.once();
 	}
 
+	@Ignore
```

Comment:
Short answer: Yes

Longer answer: Having an ignored test, that doesn't test anything specific to the current module is not really clean. So I was thinking that we could move it to the core where its more relevant.